src/main/java/net/sf/jabref/BasePanel.java

```diff
@@ -293,7 +294,7 @@ public BasePanel(JabRefFrame frame, BibtexDatabase db, File file,
                 fileMonitorHandle = Globals.fileUpdateMonitor.addUpdateListener(this,
                         file);
             } catch (IOException ex) {
-                BasePanel.logger.warning(ex.toString());
+                LOGGER.debug("Could not register FileUpdateMonitor", ex);
```

Comment:
The log levels and practices of [commons.logging](http://commons.apache.org/proper/commons-logging/guide.html#Best_Practices_General) and [java.util.logging](http://docs.oracle.com/javase/7/docs/api/java/util/logging/Level.html) are somewhat different. Since I converted the whole thing to commons, I tried to follow their guidelines. In many cases, such as the above, it is quite hard for me to decide the criticality of the event, and, hence, the log level. If you know that this event is really that critical, it is no problem to change the logging. Should I do that? Perhaps ``LOGGER.error` would also be appropriate?